CustomCommand/ImportData/Model/ResourceModel/CustomerData/Collection.php

```diff
@@ -0,0 +1,22 @@
+<?php
+namespace CustomCommand\ImportData\Model\ResourceModel\CustomerData;
+
+/**
+ * Class Collection
+ */
+class Collection extends \Magento\Framework\Model\ResourceModel\Db\Collection\AbstractCollection
+{
+    protected $_idFieldName = 'entity_id';
+    /**
+     * Init
+     */
+    protected function _construct() // phpcs:ignore PSR2.Methods.MethodDeclaration
+    {
+        $this->_init(
+            \CustomCommand\ImportData\Model\CustomerData::class,
+            \CustomCommand\ImportData\Model\ResourceModel\CustomerData::class
+        );
```

Comment:
You could have imported these two classes using the `use` statement and use alias here...